src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -59,26 +58,33 @@ const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
 
   const { SummaryView }: PluginComponents = pluginProvider.getPluginElement(pluginCode)
 
-  // TO-DO: Provide a Default SummaryView (just label/text) if SummaryView not provided in plugin
+  const DefaultSummaryView: React.FC = () => {
+    return (
+      <>
+        <label>{parameters.label}</label>
+        <Input fluid readOnly disabled transparent value={response ? response?.text : ''} />
+      </>
+    )
+  }
```

Comment:
```suggestion
  const DefaultSummaryView: React.FC = () => {
    return (
      <Form.Field required={isRequired}>
        <label>{parameters.label}</label>
        <Input fluid readOnly disabled transparent value={response ? response?.text : ''} />
      </Form.Field>
    )
  }
```

To fix the problem with the required fields

Reply:
Ah, of course! I forgot the DefaultSummaryView component wasn't with the existing <Form.Field>. Thanks 🙏